### PR TITLE
Feat: Added functionality to connect node by dragging and dropping from anywhere on node

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

Added functionality to connect node by dragging and dropping from anywhere on node
#8521
Closes [#8530](https://github.com/open-metadata/OpenMetadata/issues/8530)

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :

https://user-images.githubusercontent.com/71748675/201013606-eb6ee017-9a2a-40dd-b28c-68a1db226ee1.mov

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ const getHandle = (
           id={id}
           isConnectable={isConnectable}
           position={Position.Left}
-          style={handleStyles}
+          style={{ ...handleStyles, borderLeft: '5px solid #d9ceee', left: 0 }}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx:50]: Create a separate style object for the left and right border handles and use it here and in other places.

File: openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx [hunk 2]
```diff
@@ -128,7 +131,7 @@ const CustomNode = (props: NodeProps) => {
           { 'custom-node-header-tracing': isTraced }
         )}
         data-testid="node-label">
-        {getHandle(type, isConnectable, isNewNode)}
+        {getHandle(type, isConnectable)}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx:134]: Are we not checking for newly added nodes?

**Review by chirag-madlani — APPROVED** [review]
LGTM

**Comment by ShaileshParmar11** [comment]
Hi, @Sachin-chaurasiya comments are addressed!

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**